code/drasil-printers/drasil-printers.cabal

```diff
@@ -1,5 +1,5 @@
 Name:		drasil-printers
-Version:	0.1.0
+Version:	0.1.2
```

Comment:
Good. You should make sure that all `drasil-examples` depends on version 0.1.2 of `drasil-printers` as well.

Reply:
This has been fixed in 85d4e66d2a01eb9c79c0853e362a7056d581b09e.